docs/getting_started/install_ez_platform.md

```diff
@@ -126,6 +127,11 @@ run the following command:
     composer create-project ibexa/commerce-skeleton .
     ```
 
+!!! tip "Authentication token"
+
+    <a id="authentication-token"></a>If you added credentials to the `COMPOSER_AUTH` variable,
+    at this point add this variable to `auth.json` (for example, by running `echo COMPOSER_AUTH > auth.json`).
```

Comment:
> This can be done by redirection operator after the project is created.
> 
> ```shell
> echo COMPOSER_AUTH > auth.json
> ```
> 

```shell
echo $COMPOSER_AUTH > auth.json
```
right? :upside_down_face: